src/core/src/op/util/posponed_constant.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+#include "openvino/core/model_util.hpp"
+#include "openvino/core/graph_util.hpp"
+
+namespace ov::util {
+
+class PostponedConstant : public ov::op::Op {
+public:
+    OPENVINO_OP("Constant", "opset1");
```

Comment:
Mimic-ing `Constant` in such way is risky. Note that [`ov::is_type`](https://github.com/openvinotoolkit/openvino/blob/13df41b1e4a004e99f32c5850b7855c159fe7689/src/core/include/openvino/core/type.hpp#L96) uses this and it may lead to issues elsewhere -  [`ov::op::v0::Constant`](https://github.com/openvinotoolkit/openvino/blob/13df41b1e4a004e99f32c5850b7855c159fe7689/src/core/include/openvino/op/constant.hpp#L33) has exactly same definition.